test/lib/notifier_test.rb

```diff
@@ -0,0 +1,67 @@
+require 'test_helper'
+
+class TableTest < Minitest::Test
+  def setup
+    @bukkit               = ENV.fetch "EXPORT_BUCKET"
+    @notifier             = AuroraBootstrapper::Notifier.new(s3_path: @bukkit)
+  end
+
+  def test_export_date
+    export_date = ENV.fetch "EXPORT_DATE"
+    assert_equal export_date, AuroraBootstrapper::Notifier.new(s3_path: @bukkit).export_date
+  end
+
+  def test_region
+    region = ENV.fetch "REGION"
+    assert_equal region, AuroraBootstrapper::Notifier.new(s3_path: @bukkit).send(:region)
+  end
+
+  def test_unprefixed_path
+    assert_equal "bukkit", AuroraBootstrapper::Notifier.new(s3_path: @bukkit).send(:unprefixed_path)
```

Comment:
can we test this with a longer path name? like `s3://bukkit/something/something`

Reply:
Sure.